capstone/src/main/resources/static/savedevents.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta>
+    <title>My Account</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+<body onload="getUser();">
+    <nav>
```

Comment:
The whole nav should become an include eventually so that you don't have to duplicate it on every html page.
It will also require you to switch to server-side templating like JSP or Thymeleaf. Can you file it as a separate issue to take care of in the future?